### DISCUSSION TITLE
Camera height resets after dying in TTT

### DISCUSSION THREAD

**Discussion by Horolicht** [VickyFrenzy/gmod-enhanced-camera-2#8] [Q&A] [answered]
I have some issue with the height. Everytime a new round in Trouble in Terrorist Town starts, the height of the camera view gets a reset.
My current workaround is to apply a different playermodel, just to re-apply the playermodel I started with.
Still is there a way to store the height of the camera view, for following rounds? 
Thank you in advance and for the great addon!

**Comment by VickyFrenzy** [comment, marked as answer]
I'll try to take a look at this once I'm available.

Other addons may create conflicts. Could you dump your addon list so I can better identify what is going on?
Subscribe to [this tool](https://steamcommunity.com/sharedfiles/filedetails/?id=2443976641), start a game then run `dump_addon_list_markdown` in the [console](https://steamcommunity.com/sharedfiles/filedetails/?id=627640216). 
It will output your addon list in Markdown in the garrymod/data folder. Then you can upload it here.
If you can, make sure to remove the `.txt` in the filename so it finish with `.md` and not `.txt` when uploading it here.

**Reply by Horolicht** [reply]
The majority are maps for TTT, so I hope it's not too much to look through.
There is one addon which adjusts the height which is the "TTT - Minifier [FIXED]", but I have already tested it and even without it, I get the same issue.

[addon_list.md](https://github.com/VictorienXP/gmod-enhanced-camera-2/files/6489026/addon_list.md)

**Reply by VickyFrenzy** [reply]
Well I can confirm that I'm not experiencing this issue...
So you do have an addon that is messing with view height...
I tried to look at your addon list but you have a lot and I haven't noticed anything that could mess with the addon.
Still I can suspect it can come from a playermodel/weapon with it's own custom view height code.
If you can think of any playermodel that is not "regular" height or any other addon that might change view height...
I'll maybe try to come up with something to identify which addon(s) set the view height and/or maybe add consistency checks for the view height...

**Reply by Horolicht** [reply]
After alot of testing I found the issues. In the end it was the minifier, but not alone.
The custom Randomat is creating this issue as well. And this makes me really sad since it's one of my favorites.
Anyway thank you a lot for helping out!
